packages/@react-spectrum/calendar/src/CalendarCell.tsx

```diff
@@ -60,6 +61,15 @@ export function CalendarCell({state, currentMonth, ...props}: CalendarCellProps)
 
   let nativeDate = useMemo(() => date.toDate(state.timeZone), [date, state.timeZone]);
   let formatted = useMemo(() => dateFormatter.format(nativeDate), [dateFormatter, nativeDate]);
+  
```

Comment:
Not sure if I should move this code somewhere else, it depends on the disabled & selected state, as well as the cell's ref 🤔